.circleci/config.yml

```diff
@@ -117,7 +122,22 @@ jobs:
       - checkout
       - setup-rust
       - cargo-build
-      - run-tests
+      - run:
```

Comment:
In the PR description I wrote "Move `run-tests` command to test job, since it isn't re-used". 

I feel the main purpose of the commands is to promote reuse, but this set of instructions isn't re-used. Keeping these instructions in the job, stops maintainers from having to jump around in the config.